Makefile

```diff
@@ -34,7 +34,7 @@ deps-test:
 	bower install
 
 tools:
-	npm install bower "eslint@^2.12.0" eslint-config-edx "eslint-plugin-dollar-sign@0.0.5" "eslint-plugin-import@^1.9.2"
+	npm install bower "eslint@2.12.0" "eslint-config-edx-es5@2.0.0" "eslint-plugin-dollar-sign@0.0.5" "eslint-plugin-import@^1.9.2"
```

Comment:
Why ping "eslint@2.12.0"? "eslint@^2.12.0" is more flexible and will prevent versions conflicts.